mungegithub/mungers/submit-queue.go

```diff
@@ -486,9 +487,8 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 
 	go sq.handleGithubE2EAndMerge()
 	go sq.updateGoogleE2ELoop()
-	if sq.BatchURL != "" {
+	if sq.BatchEnabled {
```

Comment:
can you add a check earlier in this function to return an error if BatchEnabled but ProwURL is empty?